components/PlantDetails.js

```diff
@@ -55,6 +90,21 @@ export default function PlantDetails({
 
   onIsDataDefault(plant.owner === "default");
 
+  // Filter out tasks that have defined periods
+  const tasksArray = Object.entries(plant.tasks);
+  const tasksArrayFiltered = tasksArray.filter(
+    (task) => task[1].start && task[1].end
+  );
+  const tasksFiltered = Object.fromEntries(tasksArrayFiltered);
+
+  const periodColors = {
+    Seed: "#D27D2D",
+    Cultivation: "#AA336A",
+    Planting: "#79af6e",
+    Harvest: "#E23D28",
+    Pruning: "#71797E",
+  };
```

Comment:
this can be placed outside the component

Reply:
Thanks, moved.